docs/keepassxc-browser-manual.md

```diff
@@ -0,0 +1,333 @@
+---
+layout: pages
+title: KeePassXC-Browser Manual
+---
+
+This manual extends the [KeePassXC-Browser Migration](https://keepassxc.org/docs/keepassxc-browser-migration/) guide.
+
+Table of contents:
+- [1. Functionality](#1-functionality)
+    - [1.1. Accessing redentials](#11-accessing-credentials)
+        - [1.1.1. Popup](#111-popup)
+        - [1.1.2. Autocomplete](#112-autocomplete)
+        - [1.1.3. Context menu](#113-context-menu)
+        - [1.1.4. Shortcuts](#114-shortcuts)
+        - [1.2. Password generator](#12-password-generator)
+        - [1.3. Detection of credential fields](#13-detection-of-credential-fields)
+        - [1.4. Choose custom credential fields for a page](#14-choose-custom-credential-fields-for-a-page)
+        - [1.5. Create new or modify existing credentials](#15-create-new-or-modify-existing-credentials)
+        - [1.6. Choose credentials for HTTP Basic Auth requests](#16-choose-credentials-for-http-basic-auth-requests)
+        - [1.7. Fill additional values using String Fields](#17-fill-additional-values-using-string-fields)
+        - [1.8. Fill TOTP](#18-fill-totp)
+- [2. Configuration and settings](#2-configuration-and-settings)
+    - [2.1. Settings: General](#21-general)
+    - [2.2. Settings: Connected databases](#22-connected-databases)
+    - [2.3. Settings: Custom credential fields](#23-settings-custom-credential-fields)
+    - [2.4. Settings: Site preferences](#24-settings-site-preferences)
+- [3. KeePassXC browser integration settings](#3-keepassxc-browser-integration-settings)
+    - [3.1. General](#31-general)
+    - [3.2. Advanced](#32-advanced)
+- [4. Troubleshooting](#4-troubleshooting)
+
+
+### 1. Functionality
+
+This section explains the basic functionality of KeePassXC-Browser.
+
+#### 1.1. Access the credentials
+If KeePassXC-Browser detects an input field on a web page, a request is sent to KeePassXC to check if the page URL matches with any entry in the database.
+The received credentials are accesible via multiple ways which are described in the following sections.
+
+##### 1.1.1. Popup
+The extension icon background is green and it is shown with a question mark. Clicking on the icon opens a popup on where you can choose the credentials to use.
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser credentials popup" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_credentials_popup.png">
+</div>
+
+##### 1.1.2. Autocomplete
+Username field supports autocomplete for showing a list of received credentials. If there is only a password field visible on the page, autocomplete will be still available.
+By clicking the input field, the autocomplete menu is visible and correct username can be selected. Password field will be filled automatically.
```

Comment:
"and correct username" → "and the correct username". Similarly to above, "The password field" (instead of "Password field")?